war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -242,36 +242,65 @@ var FormChecker = {
   },
 
   sendRequest: function (url, params) {
```

Comment:
I realize I'm late to the party, but are we cool with breaking this function? It's used in plugins and the `response` parameter in `onComplete` having no `responseText` is a bit of a surprise to all the scripts.

Reply:
Could you clarify what's being broken? What scripts?

Reply:
Here is where I noticed it
https://github.com/jenkinsci/global-build-stats-plugin/blob/master/src/main/webapp/scripts/global-build-stats/standardFunctions.js#L39

Reply:
[JENKINS-71741](https://issues.jenkins.io/browse/JENKINS-71741) FTR